fastly-compute/geoip/src/index.js

```diff
@@ -1,8 +1,21 @@
 /// <reference types="@fastly/js-compute" />
 
 import { getGeolocationForIpAddress } from "fastly:geolocation"
+import { ConfigStore } from "fastly:config-store";
+
 async function app(event) {
   try {
+   const configStore = new ConfigStore('strings')
```

Comment:
This configures [our config_store](https://js-compute-reference-docs.edgecompute.app/docs/fastly:config-store/ConfigStore/) options from the json file configured in `fastly.toml` to check for Authentication before returning Geoip values, and error's "Unauhtorized" otherwise.